Prometheus.NetStandard/Prometheus.NetStandard.csproj

```diff
@@ -32,6 +29,11 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <!-- Enables building, but not necessarily running on non-windows systems -->
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0">
```

Comment:
adding this metapackage allows us to build the project on mac/linux hosts, which helps contributors.